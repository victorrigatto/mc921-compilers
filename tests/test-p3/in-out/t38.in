int main()
{
  char arr[2] = "str";
  return 0;
}
