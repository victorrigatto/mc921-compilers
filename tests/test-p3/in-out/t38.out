SemanticError: Size mismatch on arr initialization @ 3:8
